storage/statedb/database.go

```diff
@@ -326,15 +326,16 @@ func NewDatabaseWithCache(diskDB database.DBManager, cacheSizeMB int, daBlockNum
 	}
 	if cacheSizeMB > 0 {
 		maxEntrySizeByte := 512
+		maxEntriesInWindow := cacheSizeMB * 1024 * 1024 / maxEntrySizeByte
 		trieNodeCache, _ = bigcache.NewBigCache(bigcache.Config{
 			Shards:             1024,
 			LifeWindow:         time.Hour * 24 * 365 * 200,
-			MaxEntriesInWindow: cacheSizeMB * 1024 * 1024 / maxEntrySizeByte,
+			MaxEntriesInWindow: maxEntriesInWindow,
 			MaxEntrySize:       maxEntrySizeByte,
 			HardMaxCacheSize:   cacheSizeMB,
 			Hasher:             trieNodeHasher{},
 		})
-		logger.Debug("Initialize BigCache", "HardMaxCacheSize", cacheSizeMB, "MaxEntrySize", maxEntrySizeByte)
+		logger.Debug("Initialize BigCache", "HardMaxCacheSize", cacheSizeMB, "MaxEntrySize", maxEntrySizeByte, "MaxEntriesInWindow", maxEntriesInWindow)
```

Comment:
This is executed once. I think it can be `Info`.
```suggestion
		logger.Info("Initialize BigCache", "HardMaxCacheSize", cacheSizeMB, "MaxEntrySize", maxEntrySizeByte, "MaxEntriesInWindow", maxEntriesInWindow)
```